components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -14,27 +14,90 @@
     <div *ngIf="!loading">
       <chef-modal label="member-expression-label" class="member-expression-modal" [visible]="modalVisible" (close)="closeModal()">
         <h2>Add Member Expression</h2>
-        <div id="left-align">
+        <div id="modal-content-container">
           <form [formGroup]="expressionForm">
+
+              <chef-form-field id="create-type">
+                <label>
+                  <span class="label">Type<span aria-hidden="true">*</span></span>
+                  <select id="expression-type-dropdown"
+                          ngDefaultControl 
+                          (change)="updateFormDisplay('type')"
+                          formControlName="type">
+                    <option value='user'>User</option>
+                    <option value='team'>Team</option>
+                    <option value='token'>Token</option>
+                    <option value='*'>All members (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('type').hasError('required') && expressionForm.get('type').touched">
+                  Type is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-identity" 
+                *ngIf="expressionForm.contains('identity')"
+                [@dropInAnimation]="{ value: triggerValue, params: { height: 74.6875 }}" >
+                <label>
+                  <span class="label">Identity Provider<span aria-hidden="true">*</span></span>
+                  <select id="expression-identity-dropdown" 
+                          ngDefaultControl 
+                          (change)="updateFormDisplay('identity')"
+                          formControlName="identity">
+                    <option value="ldap">LDAP</option>
+                    <option value="local">local</option>
+                    <option value="saml">SAML</option>
+                    <option value="*">All {{ allIdentity | titlecase }}s (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('identity').hasError('required') && expressionForm.get('identity').touched">
+                  Identity Provider is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-name" 
+                *ngIf="expressionForm.contains('name')"
```

Comment:
All inputs are not being displayed based on whether or not the FormGroup contains its specific control.  This was suggested by Angular Core team as a preferred check: https://github.com/angular/angular/issues/7970#issuecomment-228624899